e2e/tests/components/radiobuttons/radiobuttons.e2e-spec.ts

```diff
@@ -124,4 +124,14 @@ describe('RadioButton Tests', () => {
         expect(await page.text1.getText()).toBe('string');
 
     });
+
+    it('should allow selection via the keyboard when no radio button is selected', async () => {

```

Comment:
There were two issues reported in the ticket, and QA feedback during the review process. Make sure you are capturing these use cases as tests!